clients/privacy-center/cypress/fixtures/consent/experience_gpp.json

```diff
@@ -16,27 +16,44 @@
             "language": "en",
             "accept_button_label": "Opt in to all",
             "acknowledge_button_label": "OK",
-            "banner_description": "We use cookies and similar methods to recognize visitors and remember their preferences. We may also use them to measure ad campaign effectiveness, target ads, and analyze site traffic. Depending on your location, you may opt-in or opt out of the use of these technologies.",
-            "banner_title": "Manage your consent preferences",
+            "banner_description": "[banner] We use cookies and similar methods to recognize visitors and remember their preferences. We may also use them to measure ad campaign effectiveness, target ads, and analyze site traffic. Depending on your location, you may opt-in or opt out of the use of these technologies.",
```

Comment:
what is this [banner] signifier for?

Reply:
Just to be sure that it's loading from `banner_title` instead of `title`, since I wanted to keep the strings consistently translated and not using fake strings as much (e.g. "This is a banner title!")

🤷 maybe I should just actually make a more obvious difference in test titles & description and do a bit of Google Translate?

Reply:
oh I see! My preference is to have the text actually be different between the two, but this is fine for now.